Automattic/pocket-casts-ios
# [Winback] New Cancel Subscription view

| 📘 Part of: #2435 
|:---:|

Fixes #2437 

This PR introduces the new Cancel Subscription screen.

| Plus | Patron |
| -------- | ------- |
| ![RocketSim_Screenshot_iPhone_16_6 1_2024-11-20_15 40 50](https://github.com/user-attachments/assets/fbe72120-31be-4fec-b13f-df56ac5c49bb) | ![RocketSim_Screenshot_iPhone_16_6 1_2024-11-20_15 41 26](https://github.com/user-attachments/assets/d5309718-b150-49db-94ac-3687d7c213d3)
 |

## To test

- CI must be 🟢 
- Make sure you have the `winback` FF on
- Use a Plus / Patron user
- Go to account and tap on Cancel Subscription
- You should see the new screen presented

## Checklist
